src/status_im/ui2/screens/chat/messages/view.cljs

```diff
@@ -118,10 +120,10 @@
                        (if platform/low-device? 700 200))))
 
 (defn get-render-data [{:keys [group-chat chat-id public? community-id admins space-keeper show-input? edit-enabled in-pinned-view?]}]
-  (let [current-public-key (<sub [:multiaccount/public-key])
-        community (<sub [:communities/community community-id])
-        group-admin? (get admins current-public-key)
-        community-admin? (when community (community :admin))
+  (let [current-public-key  (<sub [:multiaccount/public-key])
```

Comment:
Thanks for aligning the let bindings :+1:, but could you also break the long destructuring of map keys into multiple lines? It's almost wider than my screen ;)

Reply:
Good point. Nowadays the linter doesn't force us to sort namespaces. This is going to change if/when we integrate zprint in the codebase. For now, you can see there's a lot of work in this draft PR https://github.com/status-im/status-mobile/pull/14128

> I wonder If you have an Emacs shortcut for that

Emacs by itself doesn't support anything like that, but if you have installed and configured the `lsp` package https://github.com/emacs-lsp/lsp-mode, then you can call the command `lsp-clojure-clean-ns`, or define a keybinding for that (that's what I do).